sharedbuffers/mapped_struct.py

```diff
@@ -132,19 +132,28 @@
     long = int
     basestring = (bytes, str)
 
+_memoryview = cython.declare(object)
+_memoryview = memoryview
```

Comment:
This is to save us the `__Pyx_GetModuleGlobalName` stuff, right?

Reply:
Aha